aio/content/examples/schematics-for-libraries/projects/my-lib/package.json

```
@@ -0,0 +1,20 @@
+// #docplaster
+// #docregion collection
+{
+  "name": "my-lib",
+  "version": "0.0.1",
+// #enddocregion collection
+  "scripts": {
+    "build": "../../node_modules/.bin/tsc -p tsconfig.schematics.json",
+    "copy:schemas": "cp --parents schematics/*/schema.json ../../dist/my-lib/",
```

Can there be an alternative to the `--parents` flag? Can't figure out as to what to replace for this as this isn't available on macOS. (I've been trying to use `rsync` to no avail)

---

After an hour of searching and debugging, here's a better version that _should_ work cross-platform:
```suggestion
    "copy:schemas": "find schematics -type f -name 'schema.json' -exec rsync -R {} ../../dist/ngx-ytd-api-lib/ \\;",
```

---

I've submitted a PR for this: #29505

---

This will be resolved in #29505